### PR TITLE
feat: create kicker script

### PR DESCRIPTION
**Pull request by keroro520**
I redesign the kicker usage and create a new script named `kicker`. Meanwhile, I update the README.

* [README preview](https://github.com/RetricSu/godwoken-kicker/blob/kicker-script/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,68 @@
...
-```
+## Usage
 
-command to start everything:
-
-```sh
-make init
-make start
 ```
+./kicker --help
```
Comment (Flouse) [inline README.md:22]:
```suggestion
chmod +x ./kicker
./kicker --help
```

**Review by Flouse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (keroro520) [reply README.md:22]: The git-hosted `./kicker` is already executable.

**Comment by Flouse** [comment]
LGTM

**Comment by keroro520** [comment]
Update:

* 9d2062de8f7186ab12395de8960e0326be3173d8 creates workflow [`hardhat-simple-project.yml`](.github/workflows/hardhat-simple-project.yml)

**Review by RetricSu — APPROVED** [review]

File: kicker [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+	rm -rf docker/postgres/data
+	rm -rf docker/redis/data
+}
+
+function ps() {
+    compose ps "$@"
+}
+
+function logs() {
+    compose logs "$@"
```
Comment (RetricSu) [inline kicker:170]: can we have following tail logs (compose logs name -f --tail 500) ? sometimes we need to continuously watch the godwoken/web3 service activities while debug

Comment (keroro520) [reply kicker:170]: Yes, use `./kicker logs -f --tail 5000 web3`. All the command line arguments will be passed to `docker-compose`.

`./kicker logs -f --tail 5000 web3` is equal to `docker-compose -f docker/docker-compose.yml logs -f --tail 5000 web3`

**Pull request merged**